src/main/java/types/Host.java

```diff
@@ -250,6 +250,18 @@
      */
     Cpu cpu();
 
+    /**
+     * List of all host's CPUs with detailed information
+     * about the topology (socket, core) and with information
+     * about the current CPU pinning.
+     *
+     * @author Lucia Jelinkova <ljelinko@redhat.com>
+     * @since 4.5
+     * @date 15 Feb 2022
+     * @status added
+     */
+    @Link HostCpuPinning[] cpuPinnings();
```

Comment:
I wonder about the naming here - is it more about pinning or about the topology that also includes the pinning? I tend to think that if we details the physical CPU topology and add more information like where vdsm runs on (although we can also think about that as pinning) then maybe we should rather call it cpuTopology?

Reply:
I thought about it but we already have CpuTopology type in the RestAPI. To better distinguish it, I prefer the word pinning even though it is not completely accurate.

Reply:
We maybe can get closer to what we called it in the engine? `HostCpuUnit` ? just throwing an option.

Reply:
That is a good idea, I'll update the patch with a new naming.